modules/apps/forms-and-workflow/calendar/calendar-service/src/main/java/com/liferay/calendar/service/impl/CalendarBookingServiceImpl.java

```diff
@@ -497,20 +511,6 @@ public CalendarBooking invokeTransition(
 		return calendarBooking;
 	}
```

Comment:
Hey guys! @mbowerman @natocesarrego

I just changed the order of the arguments as requested by Brian Chan in https://github.com/brianchandotcom/liferay-portal/pull/44044#issuecomment-254886443. This is an interesting part of our code standards.

Thanks!